create_oc_testdata/duplicate_events/main.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.abspath('..'), "lib"))
```

Comment:
this has to be ../.. for the lib directory here

Reply:
strage... 
For me it is working with `.. `  while with `../..` it does not work.